<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>504</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Gill Sans MT Condensed</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>900</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
background-color:rgb(74, 128, 255)
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="Title">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>9</y>
       <width>891</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Hash CPU Cache</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="DataList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>321</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QListWidget" name="CacheList">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>110</y>
       <width>321</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="AddButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>140</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding:5px;
text-align: center;
border-radius:15px;
border-color:rgb(255, 255, 255)
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
     </property>
     <property name="text">
      <string>Adauga in cache</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>190</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding:5px;
text-align: center;
border-radius:15px;
border-color:rgb(255, 255, 255)
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
     </property>
     <property name="text">
      <string>Curata cache</string>
     </property>
    </widget>
    <widget class="QLabel" name="MainMemoryLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>851</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Main Memory&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="CacheMemoryLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>851</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cache Memory&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="Info">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>400</y>
       <width>521</width>
       <height>87</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Gill Sans MT Condensed</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">.QTextEdit{
background-color:rgb(74, 128, 255)
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gill Sans MT Condensed'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
